build/build.xml

```diff
@@ -254,13 +258,17 @@
   </macrodef>
 
   <!-- copy hardware folder -->
-  <target name="assemble-hardware" unless="light_bundle">
+  <target name="assemble-hardware" depends="assemble-hardware-full, assemble-hardware-light" />
+  <target name="assemble-hardware-full" unless="light_bundle">
     <copy todir="${target.path}/hardware">
       <fileset dir="../hardware">
         <exclude name="arduino/sam/**"/>
       </fileset>
     </copy>
   </target>
+  <target name="assemble-hardware-light" if="light_bundle">
+    <copy file="../hardware/package_index_bundled.json" todir="${target.path}/hardware" />
```

Comment:
I had a lot of problems with this file. Currently I add a copy fo the avr core to the hardware path with a different name and patches to use the archlinux tools. However If i define this package in this json the IDE always crashs on a name not equal to "arduino". I am not sure if the file is required in this case.

If it should be empty, maybe you should add a stub with no data in it instead. However this works for now.